scripts/kstest-runner

```diff
@@ -51,7 +51,7 @@ class VirtualInstall(object):
     Run virt-install using an iso and a kickstart
```

Comment:
Please add a commit message why you are changing `-r` to `--memory`

Reply:
Fixed.